mlflow/pyfunc/backend.py

```diff
@@ -64,8 +64,8 @@ def serve(self, model_uri, port, host):
         # NB: Absolute windows paths do not work with mlflow apis, use file uri to ensure
         # platform compatibility.
         local_uri = path_to_local_file_uri(local_path)
-        command = ("gunicorn -b {host}:{port} -w {nworkers} "
-                   "mlflow.pyfunc.scoring_server.wsgi:app").format(
+        command = ("gunicorn --timeout 60 -b {host}:{port} -w {nworkers} ${{GUNICORN_CMD_ARGS}}"
```

Comment:
This seems to be trying to extract something called GUNICORN_CMD_ARGS to the commandline, but there is no command actually extracting GUNICORN_CMD_ARGS from the os.environ.

Furthermore simply appending that string to this command is not going to override the `--timeout 60` flag which for some reason has been added back!!??!?